src/components/tasks/card/index.tsx

```diff
@@ -1,160 +1,211 @@
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'endsOn')}
-            role="button"
-            tabIndex={0}
-          >
-            {fromNowEndsOn}
-          </span>
-        </span>
-      </div>
-      <div className={classNames.cardItems}>
-        <span
-          className={classNames.cardSpecialFont}
-          contentEditable={shouldEdit}
-          onKeyPress={(e) => handleChange(e, 'startedOn')}
-          role="button"
-          tabIndex={0}
-        >
-          Started
-          {' '}
-          {fromNowStartedOn}
-        </span>
-        <span>
-          <span className={classNames.cardSpecialFont}>Assignee:</span>
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'assignee')}
-            role="button"
-            tabIndex={0}
-          >
-            {cardDetails.assignee}
-          </span>
-          <span
-            className={classNames.contributorImage}
-          >
-            <Image
-              src={assigneeProfilePic}
-              alt="Assignee profile picture"
-              onError={contributorImageOnError}
-              width={45}
-              height={45}
-            />
-          </span>
-        </span>
-      </div>
-    </div>
-  );
-};
-
-export default Card;
+import { FC, useState } from 'react';
+import Image from 'next/image';
+import classNames from '@/components/tasks/card/card.module.scss';
+import task from '@/interfaces/task.type';
+import { AVAILABLE, BLOCKED, COMPLETED, VERIFIED } from '@/components/constants/beautified-task-status';
+
+const moment = require('moment');
+
+type Props = {
+  content: task;
+  shouldEdit: boolean;
+  onContentChange: any;
+};
+
+const Card: FC<Props> = ({
+  content,
+  shouldEdit = false,
+  onContentChange = () => undefined,
+}) => {
+  const statusRedList = [BLOCKED];
+  const statusNotOverDueList = [COMPLETED, VERIFIED, AVAILABLE];
+  const cardDetails = content;
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(
+    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`,
+  );
+  const contributorImageOnError = () => setAssigneeProfilePic('/dummyProfile.png');
+
+  const localStartedOn = inputParser(cardDetails.startedOn);
+  const fromNowStartedOn = moment(localStartedOn).fromNow();
+
+  const localEndsOn = inputParser(cardDetails.endsOn);
+  const fromNowEndsOn = moment(localEndsOn).fromNow();
+  const statusFontColor = !statusRedList.includes(cardDetails.status) ? '#00a337' : '#f83535';
+  const iconHeight = '25px';
+  const iconWidth = '25px';
+
+  const cardClassNames = [classNames.card];
+
+  function isTaskOverdue() {
+    const currentDate = new Date();
+    const timeLeft = localEndsOn.valueOf() - currentDate.valueOf();

```

Comment:
Use Date.now() instead of currentDate.valueOf(), it also returns the milliseconds elapsed.